### PR TITLE
Supporting old signatures in SFRestDelegate

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

**Comment by wmathurin** [comment]
See my comment https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/72#issuecomment-331311078

File: native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/RestAPIExplorerViewController.m [hunk 1]
```diff
@@ -465,15 +465,15 @@ -(BOOL)textFieldShouldReturn:(UITextField *)textField {
 
 #pragma mark - SFRestDelegate
 
-- (void)request:(SFRestRequest *)request didLoadResponse:(id)dataResponse rawResponse:(NSURLResponse *)rawResponse {
+- (void)request:(SFRestRequest *)request didLoadResponse:(id)dataResponse {
```
Comment (wmathurin) [inline native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/RestAPIExplorerViewController.m]: for me to test ;-)
Also one should only really use the new signature if one plans to actually use the raw response

Comment (bhariharan) [reply native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/RestAPIExplorerViewController.m]: Fair enough.

**Comment by huminzhi** [comment]
## Danger has errored
[!] Invalid `DangerFile` file: `minimum_coverage_percentage` value must be a Float! Found Integer instead.
```
 #  from DangerFile:5
 #  -------------------------------------------
 #  
 >  xcov.report(
 #     scheme: 'UnitTests',
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_CI-result">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**